services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py

```diff
@@ -136,6 +149,24 @@ async def _publish_messages(self, messages: List[str]) -> None:
             aio_pika.Message(body=data.json().encode()), routing_key=""
         )
 
+    async def _publish_command(
+        self, action: str, payload: Optional[Dict[str, Any]] = None
+    ) -> None:
+        data = EventRabbitMessage(
+            node_id=self._node_id,
+            user_id=self._user_id,
+            project_id=self._project_id,
+            action=action,
+            payload=payload,
```

Comment:
if you are not using the payload. I would not add it right now. why to add complexity when we are not using it?